devtools-ci-index.yaml

```diff
@@ -99,10 +99,22 @@
     name: devtools
     jobs:
         - '{ci_project}-{git_repo}':
+            description: |
+                This job validates the syntax and yaml file used by JJB from PullReq's aginst the almighty-jobs repo
+            git_username: almighty
+            git_repo: almighty-jobs
+            ci_project: 'devtools'
+            builders:
+                - shell: |
```

Comment:
in a project the jobs section cannot be used to expand steps.

For that you'll need to either parematerize the job-template or create a new job-template to use here.